src/anna/week_2/Finra.java

```diff
@@ -0,0 +1,34 @@
+package anna.week_2;
+
+public class Finra {
+    public static void main(String[] args) {
+        int a = 30;
+        finra(a);
+
+    }
+    public static void finra(int num) {
+        for (int i = 1; i <= num; i++) {
+            if (i % 3 == 0 && i % 5 == 0) {
+                System.out.print("FINRA" + " ");
+
+            } else if (i % 3 == 0) {
+                System.out.print("FIN" + " ");
+
+            } else if (i % 5 == 0) {
+                System.out.print("RA" + " ");
+
+            }
```

Comment:
Hi, Anna
It would be best if you had to insert  this print statement System.out.print(i + " ");
inside else statement, otherwise it prints FINRA and 30 in the end